src/index.js

```diff
@@ -80,6 +81,7 @@ async function runner(ctx, name) {
 			}
 		}
 	} finally {
+		state.__test__ = '';
```

Comment:
if there is no name, reckon we just make this `null`?

Reply:
For performance & value manipulation later, better to keep types the same.

Reply:
knew there'd be a "lukeed" reason for it ;) haha brilliant!